pkgs/development/python-modules/python-snappy/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, python, snappy, cffi }:
+
+buildPythonPackage rec {
+  pname = "python-snappy";
+  version = "0.5.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0jnp4xxaif585l76nqf3fviva4iclgdd51sksl342g880qzq0zwa";
+  };
+
+  propagatedBuildInputs = [ snappy cffi ];
+
+  preCheck = ''
+    export PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Python bindings for the snappy google library";
+    homepage = "https://github.com/andrix/${pname}";
```

```suggestion
    homepage = "https://github.com/andrix/python-snappy";
```
I tend to not do this so people can visit the link easily when reading the code.

---

@worldofpeace: I agree. Re-added it.